### PR TITLE
Add better support for HP header binary data

### PR DESCRIPTION
**Pull request by thliebig**
Signed-off-by: Thorsten Liebig liebig@imst.de

### CODE REVIEW

**Comment by thliebig** [comment]
In binary format my device a "HP 8719D" was truncating binary read-out on encountering the '\n' byte in the data stream.  
According to the document "8719D, 8720D, and 8722D Network Analyzer Programmer's Guide (Feb99)", page 34 (1-23) "Array-Data Formats", this change allows data formats "FORM2", "FORM3" and "FORM5" to be read correctly.

**Comment by MatthieuDartiailh** [comment]
LGTM but could you add a small test for the parsing function @thliebig

**Comment by MatthieuDartiailh** [comment]
Now that bors is working, a test is really all that is missing.

File: pyvisa/util.py [hunk 1]
```diff
@@ -296,6 +296,29 @@ def parse_ieee_block_header(block):
 
     return offset, data_length
 
+def parse_hp_block_header(block, is_big_endian):
+    """Parse the header of a HP block.
+
+    Definite Length Arbitrary Block:
+    #A<data_length><data>
+
+    The header ia always 4 bytes long.
```
Comment (MatthieuDartiailh) [inline pyvisa/util.py]: typo ia->is

**Comment by MatthieuDartiailh** [comment]
Could you please add a test ?

**Comment by MatthieuDartiailh** [comment]
Closed by #301

**Pull request closed without merging**